packages/klevu-ui/src/components/klevu-quicksearch/klevu-quicksearch.tsx

```diff
@@ -226,6 +232,11 @@ export class KlevuQuicksearch {
       this.trendingProducts = resultObject.records
       this.#emitChangedData()
     }
+
+    const settings = getKMCSettings()
+    if (this.usePersonalisation === undefined && settings?.klevu_uc_userOptions.enablePersonalisationInSearch) {
+      this.usePersonalisation = true
```

Comment:
would it be better to have it as
this.usePersonlization = settings?.klevu_uc_userOptions.enablePersonalisationInSearch || false
so that it always is set with a boolean value true/false.